winbooks-api-extra/src/main/java/be/valuya/winbooks/api/accountingtroll/converter/ATBookPeriodConverter.java

```diff
@@ -26,6 +26,16 @@ public ATBookPeriod convertToTrollPeriod(WbBookYearFull wbYear, WbPeriod wbPerio
         LocalDate periodEndDate = wbPeriod.getEndDate();
         ATPeriodType accountingPeriodType = getAccountingPeriodType(wbPeriod);
 
+        if (accountingPeriodType == ATPeriodType.OPENING) {
+            LocalDate yearStartDate = wbYear.getStartDate();
+            periodStartDate = yearStartDate;
+            periodEndDate = yearStartDate;
+        } else if (accountingPeriodType == ATPeriodType.CLOSING) {
+            LocalDate yearEndDate = wbYear.getEndDate();
```

Comment:
I expect endDate to be a strict upper bound, right? e.g. start = 2018/1/1 ; end = 2019/1/1